client/src/modules/stock/inventories/templates/inventory.action.html

```diff
@@ -14,5 +14,17 @@
         <span class="fa fa-file-text-o"></span> <span translate>TREE.STOCK_MOVEMENTS</span>
       </a>
     </li>
+  
+    <li>
+      <a data-method="inventories" href
+        ui-sref="inventory.list({ 
+          uuid :  row.entity.inventory_uuid,
+          created : false,
+          updated : false,
+          filters : [
+          { key : 'text', value : row.entity.text }]})">
```

Comment:
Let's search based on UUID rather than on text.  Like this:
```html
<a ui-sref="inventory.list({ filters : [{ key : 'uuid', value : row.entity.inventory_uuid, displayValue: row.entity.text, cacheable: false }]})">
  ...
</a>

````

Reply:
Note that, you'll need to add `uuid` to the list of registered filters in the [InventoryService](https://github.com/IMA-WorldHealth/bhima/blob/master/client/src/modules/inventory/inventory.service.js#L115).